recipes/portable-file-dialogs/all/conanfile.py

```diff
@@ -0,0 +1,31 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class PortableFileDialogsConan(ConanFile):
+    name = "portable-file-dialogs"
+    license = "WTFPL"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/samhocevar/portable-file-dialogs"
+    description = "Portable GUI dialogs library, C++11, single-header"
+    topics = ("conan", "gui", "dialogs")
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
```

Comment:
```suggestion

    def configure(self):
        if self.settings.compiler.get_safe("cppstd"):
            tools.check_min_cppstd(self, 11)

```

Reply:
is it really needed? it works without this